<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DroneProcDlg</class>
 <widget class="QDialog" name="DroneProcDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Process Drone Imagery</string>
  </property>
  <widget class="QPushButton" name="cmdRun">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>111</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdCancel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>310</y>
     <width>111</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Initial Stage:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>184</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Outputs:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkDTM">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>188</y>
     <width>271</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Ground points (lowest returns)">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt; Use this tag to build a DTM (Digital Terrain Model, ground only) using a simple morphological filter.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>DTM - Digital Terrain Model (ground)</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkDSM">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>208</y>
     <width>271</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Surface points (highest returns)">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Use this tag to build a DSM (Digital Surface Model, ground + objects) using a progressive morphological filter.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>DSM - Digital Surface Model (canopy)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>48</y>
     <width>129</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Crop Buffer:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinCrop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>48</y>
     <width>62</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Automatically crop image outputs by creating a smooth buffer around the dataset boundaries, shrunk by N meters. Use 0 to disable cropping.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QComboBox" name="cboInitStage">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>16</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>72</y>
     <width>129</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>DEM Resolution:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinDemResolution">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>72</y>
     <width>62</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;DSM/DTM resolution in cm/pixel. Note that this value is capped by a ground sampling distance (GSD) estimate.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>96</y>
     <width>129</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Orthophoto Resolution:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinOrthoResolution">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>96</y>
     <width>62</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Orthophoto resolution in cm/pixel. Note that this value is capped by a ground sampling distance (GSD) estimate.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="maximum">
    <double>1000.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinGapFill">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>120</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Number of steps used to fill areas with gaps. Set to 0 to disable gap filling. Starting with a radius equal to the output resolution, N different DEMs are generated with progressively bigger radius using the inverse distance weighted (IDW) algorithm and merged together. Remaining gaps are then merged using nearest neighbor interpolation.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>120</y>
     <width>129</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>DEM Gap Fill Steps:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>304</x>
     <y>16</y>
     <width>209</width>
     <height>153</height>
    </rect>
   </property>
   <property name="title">
    <string>Point Cloud Options</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>72</y>
      <width>129</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkPcClassify">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>24</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Classify the point cloud outputs. You can control the behavior of this option by tweaking the --dem-* parameters.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Classify</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkPcRectify">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>48</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Perform ground rectification on the point cloud. This means that wrongly classified ground points will be re-classified and gaps will be filled. Useful for generating DTMs.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Rectify</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>96</y>
      <width>129</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sample:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>120</y>
      <width>129</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Quality:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboPcQuality">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>120</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set point cloud quality. Higher quality generates better, denser point clouds, but requires more memory and takes longer. Each step up in quality increases processing time roughly by a factor of 4x.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinPcFilter">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>72</y>
      <width>62</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Filters the point cloud by removing points that deviate more than N standard deviations from the local mean. Set to 0 to disable filtering.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinPcSample">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>96</y>
      <width>62</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Filters the point cloud by keeping only a single point around a radius N (in meters). This can be useful to limit the output resolution of the point cloud and remove duplicate points. Set to 0 to disable sampling.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000000000000000</double>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cboInitStage</tabstop>
  <tabstop>spinCrop</tabstop>
  <tabstop>spinDemResolution</tabstop>
  <tabstop>spinOrthoResolution</tabstop>
  <tabstop>spinGapFill</tabstop>
  <tabstop>chkDTM</tabstop>
  <tabstop>chkDSM</tabstop>
  <tabstop>cmdRun</tabstop>
  <tabstop>cmdCancel</tabstop>
  <tabstop>chkPcClassify</tabstop>
  <tabstop>chkPcRectify</tabstop>
  <tabstop>spinPcFilter</tabstop>
  <tabstop>spinPcSample</tabstop>
  <tabstop>cboPcQuality</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
